environment.yml

```diff
@@ -12,24 +12,25 @@ dependencies:
       - git+https://github.com/analysiscenter/radio.git@6d53e25#egg=radio
       - azure-mgmt-resource==12.1.0
       - azure-mgmt-datafactory==1.1.0
-      - azureml-mlflow==1.23.0
-      - azureml-sdk==1.23.0
-      - azureml-tensorboard==1.23.0
+      - azureml-mlflow==1.32.0
+      - azureml-sdk==1.32.0
+      - azureml-tensorboard==1.32.0
       - conda-merge==0.1.5
       - dataclasses-json==0.5.2
       - docker==4.3.1
       - flake8==3.8.3
       - gitpython==3.1.7
       - gputil==1.4.0
       - h5py==2.10.0
+      - hi-ml
```

Comment:
fix version or always latest? Do we need azureml-sdk here if hi-ml already depends on it?

Reply:
We may not need azureml-sdk, but it is wise for us to fix the version for the azureml-sdk. I'm letting the version of hi-ml sty unpinned since there are so many updates planned over the next few weeks, but perhaps that's unwise? Perhaps we should run some InnerEye tests as part of the HI-ML PR build so we know if something's gone awry?

Reply:
Pinned